test/message/source_map_sourcemapping_url_string.out

```diff
@@ -0,0 +1,3 @@
+Error: an exception.
+    at *typescript-sourcemapping_url_string.ts:3:7*
+    at Module._compile (node:internal/modules/cjs/loader:1129:14)
```

Comment:
Line numbers and column numbers of the sources that are not the test fixtures should be matched with `*`:

```suggestion
    at Module._compile (node:internal/modules/cjs/loader:*)
```